<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Logon">
     <attribute name="title">
      <string>Logon</string>
     </attribute>
     <widget class="QTableWidget" name="logonTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>791</width>
        <height>531</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="Service">
     <attribute name="title">
      <string>Service</string>
     </attribute>
     <widget class="QTableWidget" name="ServiceTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>791</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Drivers">
     <attribute name="title">
      <string>Driver</string>
     </attribute>
     <widget class="QTableWidget" name="DriversTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Sched task </string>
     </attribute>
     <widget class="QTableWidget" name="TaskTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>KnownDlls</string>
     </attribute>
     <widget class="QTableWidget" name="DllTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
